docs/source/getting-started/content/proc_upgrading-codeready-containers.adoc

```diff
@@ -0,0 +1,31 @@
+[id="upgrading-codeready-containers_{context}"]
+= Upgrading {prod}
+
+Newer versions of the {prod} binary require manual set up to prevent potential incompatilibities with previous versions.
+
+.Procedure
+
+* link:{crc-download-url}[Download the latest release of {prod}].
+
+* Delete the existing {prod} virtual machine:
++
+[subs="+quotes,attributes"]
+----
+$ {bin} delete
+----
+
+* Replace the previous [command]`{bin}` binary with the binary of the latest release.
+
```

Comment:
@robin-owen can we add here to make sure the user is using correct binary by adding the `crc version` command ?